frappe/public/js/frappe/ui/toolbar/fuzzy_match.js

```diff
@@ -81,8 +81,18 @@ function fuzzy_match_recursive(
 	// Loop through pattern and str looking for a match.
 	let first_match = true;
 	while (pattern_cur_index < pattern.length && str_curr_index < str.length) {
+		// Normalize and compare individual characters
+		const normalizedPatternChar = pattern[pattern_cur_index]
+			.normalize("NFD")
+			.replace(/[\u0300-\u036f]/g, "")
+			.toLowerCase();
+		const normalizedStrChar = str[str_curr_index]
+			.normalize("NFD")
+			.replace(/[\u0300-\u036f]/g, "")
+			.toLowerCase();
+
```

Comment:
```suggestion
		const normalized_pattern_char = pattern[pattern_cur_index]
			.normalize("NFD")
			.replace(/[\u0300-\u036f]/g, "")
			.toLowerCase();
		const normalized_str_char = str[str_curr_index]
			.normalize("NFD")
			.replace(/[\u0300-\u036f]/g, "")
			.toLowerCase();
```
we use snake_case instead of camelCase

Reply:
You’re right, I‘m sorry, but thanks for fixing! 🙏
We might actually want to add the snake_case rule to our Linter?